shared/helpers.ts

```diff
@@ -137,6 +137,41 @@ const isSDKCloned = () => {
   return fs.existsSync(getSDKPath())
 }
 
+const copyFileSync = (source, target) => {
```

Comment:
Should we specify the data types of `source` and `target` here?

Reply:
Yeah it should. I'll add the types there.